### PR TITLE
Style charts.

### PR DESCRIPTION
**Pull request by nevingeorge**
Style the compare page charts.

image

### CODE REVIEW

File: DSA-Campaign-Uplift-Estimation/src/main/webapp/Compare/script-compare.js [hunk 1]
```diff
@@ -108,32 +116,112 @@ function drawBarGraph() {
...
+    headersRow.appendChild(header4);
+    headersRow.appendChild(header5);
+    headersRow.appendChild(header6);
+    headersRow.appendChild(header7);
+    headersRow.appendChild(header8);
+    headersRow.appendChild(header9);
+    headersRow.appendChild(header10);
+    headersRow.appendChild(header11);
+    headersRow.appendChild(header12);
+    headersRow.appendChild(header13);
```
Comment (avenkat-google) [inline DSA-Campaign-Uplift-Estimation/src/main/webapp/Compare/script-compare.js:166]: This is a lot of repeated code. Can we have a function that takes in a list of header names?

File: DSA-Campaign-Uplift-Estimation/src/main/webapp/Compare/script-compare.js [hunk 2]
```diff
@@ -108,32 +116,112 @@ function drawBarGraph() {
...
+        rowElements.appendChild(element4);
+        rowElements.appendChild(element5);
+        rowElements.appendChild(element6);
+        rowElements.appendChild(element7);
+        rowElements.appendChild(element8);
+        rowElements.appendChild(element9);
+        rowElements.appendChild(element10);
+        rowElements.appendChild(element11);
+        rowElements.appendChild(element12);
+        rowElements.appendChild(element13);
```
Comment (avenkat-google) [inline DSA-Campaign-Uplift-Estimation/src/main/webapp/Compare/script-compare.js:215]: Same here

File: DSA-Campaign-Uplift-Estimation/src/main/webapp/Compare/script-compare.js (hunk 1 above)
Comment (nevingeorge) [reply DSA-Campaign-Uplift-Estimation/src/main/webapp/Compare/script-compare.js]: Sorry done, this should have been obvious.

File: DSA-Campaign-Uplift-Estimation/src/main/webapp/Compare/script-compare.js (hunk 2 above)
Comment (nevingeorge) [reply DSA-Campaign-Uplift-Estimation/src/main/webapp/Compare/script-compare.js]: Fixed

**Pull request merged**